proposals/compatibility-mode.md

```diff
@@ -101,19 +101,18 @@ The differences in limits between compatibility mode and standard WebGPU
 are as follows
 
 
-| limit                               | compat  | standard  |
-| :---------------------------------- | ------: | --------: |
-| `maxColorAttachments`               |       4 |         8 |
-| `maxComputeInvocationsPerWorkgroup` |     128 |       256 |
-| `maxComputeWorkgroupSizeX`          |     128 |       256 |
-| `maxComputeWorkgroupSizeY`          |     128 |       256 |
-| `maxInterStageShaderVariables`      |      15 |        16 |
-| `maxStorageBuffersPerShaderStage`   |       4 |         8 |
-| `maxTextureDimension1D`             |    4096 |      8192 |
-| `maxTextureDimension2D`             |    4096 |      8192 |
-| `maxTextureDimension3D`             |    2048 |      2048 |
-| `maxUniformBufferBindingSize`       |   16384 |     65536 |
-| `maxVertexAttributes`        | 16<sup>a</sup> |        16 |
+| limit                               | compat  | standard  | gl limit                                     |
+| :---------------------------------- | ------: | --------: | :------------------------------------------- |
+| `maxColorAttachments`               |       4 |         8 | min(MAX_COLOR_ATTACHMENTS, MAX_DRAW_BUFFERS) |
+| `maxComputeInvocationsPerWorkgroup` |     128 |       256 | MAX_COMPUTE_WORK_GROUP_INVOCATIONS           |
+| `maxComputeWorkgroupSizeX`          |     128 |       256 | MAX_COMPUTE_WORK_GROUP_SIZE                  |
+| `maxComputeWorkgroupSizeY`          |     128 |       256 | MAX_COMPUTE_WORK_GROUP_SIZE                  |
+| `maxInterStageShaderVariables`      |      15 |        16 | MAX_VARYING_VECTORS                          |
+| `maxStorageBuffersPerShaderStage`   |       4 |         8 | MAX_SHADER_STORAGE_BUFFER_BINDINGS           |
```

Comment:
[`GL_MAX_SHADER_STORAGE_BUFFER_BINDINGS`](https://opengles.gpuinfo.org/displaycapability.php?name=GL_MAX_SHADER_STORAGE_BUFFER_BINDINGS&esversion=31) seems to be 8.

[`GL_MAX_VERTEX_SHADER_STORAGE_BLOCKS`](https://opengles.gpuinfo.org/displaycapability.php?name=GL_MAX_VERTEX_SHADER_STORAGE_BLOCKS&esversion=31) & [`GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS`](https://opengles.gpuinfo.org/displaycapability.php?name=GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS&esversion=31) are 4 though.

We probably need something like `min(GL_MAX_SHADER_STORAGE_BUFFER_BINDINGS, GL_MAX_VERTEX_SHADER_STORAGE_BLOCKS, GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS, GL_MAX_COMPUTE_SHADER_STORAGE_BLOCKS)` 😅.